app/assets/stylesheets/govuk-component/_metadata.scss

```diff
@@ -9,13 +9,16 @@
   }
 
   dt {
-    float: left;
-    clear: left;
-    width: auto;
-    min-width: 120px;
+    margin-top: 0.5em;
+    line-height: normal;
 
     @include media(tablet) {
+      float: left;
+      clear: left;
+      width: calc(30% - 10px);
```

Comment:
No, we could use border-box. That would just mean two lines of CSS instead of one.